devops/README.md

```diff
@@ -59,3 +59,11 @@ Your script will grab devopsshared_0*_file1.txt, file3.txt and file5.txt, then i
 
 # Resources
 TBD
```

Comment:
@Nkoli You can remove line 61 since you have content added already.

Reply:
Done.